README.md

```diff
@@ -91,6 +91,9 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `java.debug.settings.maxStringLength`: the maximum length of string displayed in "Variables" or "Debug Console" viewlet, the string longer than this length will be trimmed, defaults to `0` which means no trim is performed.
 - `java.debug.settings.enableHotCodeReplace`: enable hot code replace for Java code. Make sure the auto build is not disabled for [VSCode Java](https://github.com/redhat-developer/vscode-java). See the [wiki page](https://github.com/Microsoft/vscode-java-debug/wiki/Hot-Code-Replace) for more information about usages and limitations.
 
+## Troubleshooting
+Reference the [troubleshooting guide](https://github.com/Microsoft/vscode-java-debug/blob/master/Troubleshooting.md) for the frequently asked errors.
```

Comment:
errors ==> issues  

It doesn't mean errors in this context. User may just want to know more about the information, warning and errors in these cases.